src/util/AttachedPlayer.cpp

```diff
@@ -0,0 +1,61 @@
+#include "AttachedPlayer.hpp"
+
+using namespace chatterino;
```

Comment:
Should stick this into the chatterino namespace instead of having to use `using namespace`